pilot/pkg/networking/util/util.go

```diff
@@ -246,6 +246,15 @@ func IsProxyVersionGE11(node *model.Proxy) bool {
 	return ver >= "1.1"
 }
 
+func IsXDSMarshallingToAnyEnabled(node *model.Proxy) bool {
+	// This feature previously supposed to launch with v1.1, however due to a regression that was found last minute
+	// ahead of the release this needs to be turned off for v1.1, hence introducing a separate feature flag for
+	// this (disabled by default).
+
+	// return IsProxyVersionGE11(node)
+	return false
```

Comment:
default to true and override with config please

Reply:
oh, and you probably have to preserve the old IsProxyVersionGE11 check too.  if that is false then the config override would be ignored / never use Any for older proxies that don't understand it no matter what the config says